### PR TITLE
[aapcs64] Fix typos

### PR DESCRIPTION
**Pull request by GuillermoCallaghan**
This PR fixes typos and some formatting issues

### CODE REVIEW

**Comment by rsandifo-arm** [comment]
Thanks for the patch.  It mostly looks good.

It's unfortunate that the current capitalisation choices are inconsistent.  However, we'd like to move in the opposite direction from the patch and use Sentence case rather than Title Case for section headings.  Perhaps there's an exception to be made for things like Homogeneous Floating-point Aggregates, where the capitalisation follows the abbreviation (HFA).  (“Homogeneous Floating-point Aggregates” is used with that capitalisation within the text body too.)

**Review by brandingfuture — APPROVED** [review]

**Comment by GuillermoCallaghan** [comment]
Thanks for the feedback. I have reverted the capitalisation of some of the
titles, and changed the other to Sentence case.

I left these terms unchanged:

* Homogeneous Aggregates
* Pure Scalable Types

## Inconsistencies

These are some terms in witch I found inconsistencies.
If you can point me which are the preferred ones I can change them in a commit.

* Floating Point
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | Floating Point        | 10           |
  | Floating point        | 1            |
  | floating point        | 5            |

  Also Floating-Point:

  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | Floating-Point        | 10           |
  | Floating-point        | 11           |
  | floating point        | 25           |

* Scalable predicate
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | Scalable Predicate    | 13           |
  | scalable predicate    | 11           |

* scalable vector types
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | scalable vector       | 17           |
  | Scalable Vector       | 15           |
  | Scalable vector       |  4           |

* Fundamental Data Type
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | Fundamental Data Type | 13           |
  | fundamental data type | 6            |

* composite type
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | composite type        |  7           |
  | Composite Type        | 13           |

* Natural Alignment
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | natural alignment     |  6           |
  | Natural Alignment     |  4           |

* Frame Pointer
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | frame pointer         |  6           |
  | Frame Pointer         |  3           |

* link register
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | link register         |  2           |
  | Link Register         |  1           |

* stack pointer
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | stack pointer         |  4           |
  | Stack Pointer         |  1           |

* Arm Alternative Format
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | Arm Alternative Format|  2           |
  | Arm Alternative format|  3           |

* Arm Alternative Format
  |   Capitalisation      | Repetitions  |
  |-----------------------|-------------:|
  | short vector/s        |  21          |
  | Short Vector/s        |  3           |
  | Short vector/s        |  4           |

**Comment by stuij** [comment]
@philgee-oss @sallyarmneale : could you review this patch and see if you agree this is the right thing to do for the ABI documents? Thanks!

File: aapcs64/aapcs64.rst [hunk 1]
```diff
@@ -341,16 +341,16 @@ VG
...
 
 LP64
    SysV-like data model where int is 32-bit, but long int and pointer are 64-bit.
 
 LLP64
    Windows-like data model where int and long int are 32-bit, but long long int and pointer are 64-bit.
 
-This document uses the following terms.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This Document Uses the Following Terms
```
Comment (philgee-oss) [inline aapcs64/aapcs64.rst]: This should not be a title, but an ordinary sentence.

This document uses the following abbreviations:

Comment (GuillermoCallaghan) [reply aapcs64/aapcs64.rst]: Done. 1a02375

**Review by philgee-oss — APPROVED** [review]
LGTM

**Pull request merged**